scripts/tuple_tree_generator/tuple_tree_generator/templates/tuple_tree_gen.py.tpl

```diff
@@ -28,6 +28,9 @@ from revng.tupletree import YamlDumper as _ExternalYamlDumper
 from .external import 't'
 ## endfor ##
 
+##- for import in get_mixins_imports() ##
+'import'
```

Comment:
`'` as jinja delimiter doesn't sound like the best.
Can we use something more conventional?
Low-priority, feel free to append a commit at the end.

Reply:
Replaced with `#{` and `}#`